applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_eigensolver.py

```diff
@@ -78,7 +78,7 @@ def _create_mechanical_solution_strategy(self):
         if solver_type == "eigen_eigensystem":
             mass_matrix_diagonal_value = 0.0
             stiffness_matrix_diagonal_value = 1.0
-        elif solver_type == "feast" or solver_type == "eigen_feast":
+        elif solver_type == "eigen_feast":
```

Comment:
since now the feast in the eigensolversapp is the only one I would use the name `feast`
```suggestion
        elif solver_type == "feast":
```

Reply:
can you please update the remaining occurences too?